sympy/utilities/runtests.py

```diff
@@ -225,6 +225,9 @@ def run_in_subprocess_with_hash_randomization(
 
     function_kwargs = function_kwargs or {}
 
+    # Add path to the git sympy directory to PYTHONPATH envirnoment variable
+    # before importing sympy
+    sys.path.insert(0, "../../.")
```

Comment:
../../. is relative to the user's current directory, not the location of this file. You need to use `__file__` to get the directory of this file.